src/util/find_uid.c

```diff
@@ -365,9 +443,9 @@ errno_t check_if_uid_is_active(uid_t uid, bool *result)
     /* fall back to the old method */
```

Comment:
(not introduced by this patch but)
either I don't understand something, or there is missing `return EOK;` after `*result = false;` above.

Reply:
Yes, we are missing `return EOK;` in the no active session condition